src/Forms/Validation/RequiredFieldsValidator.php

```diff
@@ -15,7 +18,7 @@
  *
  * @deprecated 5.4.0 Will be renamed to SilverStripe\Forms\Validation\RequiredFieldsValidator
```

Comment:
```suggestion
```
Please also remove any other deprecation notices that were done in the CMS 5 round for this card.

Reply:
Updated